recipes/sharedtags-greedyview.lua

```diff
@@ -0,0 +1,195 @@
+--- Provides functionality to share tags across all screens in awesome WM.
+-- @module sharedtags
+-- @author Albert Diserholt
+-- @copyright 2016 Albert Diserholt
+-- @license MIT
+
+-- Grab environment we need
+local awful = require("awful")
+local capi = {
+    screen = screen
+}
+
+local sharedtags = {
+    _VERSION = "sharedtags v1.0.0 for v4.0",
+    _DESCRIPTION = "Share tags for awesome window manager v4.0 with greedyview",
+    _URL = "https://github.com/dakling/dakling/awesome-www",
```

Comment:
Oh and: Is it a good idea to leave the URL in? It seems like this file was copied around and possibly changed at least four times since it was on that URL.